### PR TITLE
Delete litmus crd's during the cleanup

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit will ensure that the litmus resources installed on the
cluster get cleaned up and also creates the chaosengine in the
specified namespace.

### CODE REVIEW

**Comment by paigerube14** [comment]
2 thoughts in general with all the litmus scenarios: 

* If the namespace that you set in the config doesn't match the namespace specified in the engine file, an error gets printed. I think this could be really problematic for the user copying over there own litmus scenarios and not properly setting the namespace in the engine file and config. Not 100% sure how to handle this situation though. **Thoughts**? 

`error: the namespace from the provided object "litmus-test" does not match the namespace "litmus". You must pass '--namespace=litmus-test' to perform this operation.`


* Should we put in an error if we read in <node_name> in any of the engine yaml files to remind the user to fill them in before running.

**Review by paigerube14 — APPROVED** [review]
This PR itself looks good to me!

**Comment by chaitanyaenr** [comment]
> 2 thoughts in general with all the litmus scenarios:
> 
>     * If the namespace that you set in the config doesn't match the namespace specified in the engine file, an error gets printed. I think this could be really problematic for the user copying over there own litmus scenarios and not properly setting the namespace in the engine file and config. Not 100% sure how to handle this situation though. **Thoughts**?
> 
> 
> `error: the namespace from the provided object "litmus-test" does not match the namespace "litmus". You must pass '--namespace=litmus-test' to perform this operation.`
> 
>     * Should we put in an error if we read in <node_name> in any of the engine yaml files to remind the user to fill them in before running.

You are right @paigerube14, thinking more about this it might be better to not have the ability to configure the namespace given that we would need to create the namespace and manage it if it doesn't exist and ask the user to switch the namespace in the chaosengine to litmus in case of bringing their own configs like you mentioned. Latest commit should have the changes, please take a look and let me know if you think this helps address it. Thanks.

File: kraken/litmus/common_litmus.py [hunk 1]
```diff
@@ -25,6 +25,13 @@ def run(scenarios_list, config, litmus_uninstall, wait_duration, litmus_namespac
                 if yaml_item["kind"] == "ChaosEngine":
                     engine_name = yaml_item["metadata"]["name"]
                     experiment_names = yaml_item["spec"]["experiments"]
+                    experiment_namespace = yaml_item["metadata"]["namespace"]
+                    if experiment_namespace != "litmus":
+                        logging.error(
+                            "Specified namespace in the scenario: %s is not supported, please switch it to litmus"
```
Comment (paigerube14) [inline kraken/litmus/common_litmus.py:31]: Can you add what file the incorrect namespace is in, might be easier to find that way

Comment (chaitanyaenr) [reply kraken/litmus/common_litmus.py]: Done. PTAL.

**Review by paigerube14 — APPROVED** [review]
LGTM

**Pull request merged**